packages/devtools-plugin/tests/utils/redux-connector.ts

```diff
@@ -11,8 +11,11 @@ export class ReduxDevtoolsMockConnector {
   private dispatcher: Subject<NgxsDevtoolsAction> = new Subject<NgxsDevtoolsAction>();
   private countId = 0;
 
-  public subscribe(fn: Function): Subscription {
-    return this.dispatcher.subscribe(e => fn(e));
+  public subscribe(fn: Function): VoidFunction {
+    const subscription = this.dispatcher.subscribe(e => fn(e));
+    return () => {
+      return subscription.unsubscribe();
+    };
```

Comment:
What is the reason for this extra code and slight signature change?

Reply:
This is the correct signature. The actual redux connector doesn't return a subscription but rather a `VoidFunction`.